### PR TITLE
StepanSr: Homework TaskGenerics

### PR DESCRIPTION
**Pull request by StepanSrr**

### CODE REVIEW

File: 02-GenericsAndLambda/src/main/java/homework/Stepan/TaskGenerics.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        return object1;
+    }
+
+    public T2 getSecond() {
+        return object2;
+    }
+}
+
+class PairTest {
+    public static void main(String[] args) {
```
Comment (kykaparov) [inline 02-GenericsAndLambda/src/main/java/homework/Stepan/TaskGenerics.java:48]: I would put this class in a separate file. Generally, it is code practice to have one class in one file. And here you have two main methods, that is not good :)

Comment (StepanSrr) [reply 02-GenericsAndLambda/src/main/java/homework/Stepan/TaskGenerics.java]: @kykaparov done

**Comment by kykaparov** [comment]
This PR is ready to be merged

**Comment by kykaparov** [comment]
Before merging could you squash commits to one commit?

**Pull request merged**